<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailDialogUi</class>
 <widget class="QDialog" name="MailDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TurnFix - E-Mails versenden</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/TurnFix.qrc">
    <normaloff>:/appicons/icon.png</normaloff>:/appicons/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Absender:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Betreff:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="txt_from">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="txt_subject"/>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QPlainTextEdit" name="txt_to">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QListWidget" name="lst_attach">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Vereinsmeldeliste</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Meldeliste</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Meldematrix</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Riegenlisten</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ergebnisliste mit Details</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ergebnisliste ohne Details</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vereinsergebnisliste mit Details</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vereinsergebnisliste ohne Details</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rudengesamtwertung</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zeitplan</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Anh√§nge:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="3" colspan="2">
    <widget class="QTextEdit" name="txt_text"/>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="but_send">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>versenden</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="but_to">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>An:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
